app.js

```diff
@@ -33,8 +32,10 @@ app.use(require('express-session')({
 app.use(express.static(__dirname + '/public/static/'));
 app.use(express.static(__dirname + '/public/angular'));
 
+app.get(/^\/(index)?$/, function(req, res) {
```

Comment:
Comment 1:
It will be better if this could match '/' or 'index' or 'index.html', so I suggest modify it a little bit:
`/^\/(index(.html)?)?$/`